phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/engines/EnginesController.java

```diff
@@ -64,7 +65,7 @@ public EnginesController(ViewEngines viewEngines, RepositoryManager repositoryMa
         this.themeManager = themeManager;
 
         this.repositoryManager.addCallbacks(this::populateView,
-                e -> Platform.runLater(() -> viewEngines.showFailure()));
+                e -> Platform.runLater(() -> viewEngines.showFailure(tr("Loading engines failed."), Optional.of(e))));
```

Comment:
Just asking:
Am I overlooking something or are you always using `Optional.of(e)`? If this is the case why are we using an `Optional`, `Optional.of(null)` will throw an error?